### PR TITLE
Added our updated management client with the updated cloudcredential Client

### PR DESCRIPTION
**Pull request by igomez06**

### CODE REVIEW

**Review by paynejacob — CHANGES_REQUESTED** [review]
discussed this offline, we are going to take a different approach

**Comment by igomez06** [comment]
> discussed this offline, we are going to take a different approach

new changes are up now.

File: tests/framework/extensions/cloudcredentials/config.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package cloudcredentials
+
+import (
+	"github.com/rancher/norman/types"
+)
+
+const CloudCredentialType = "cloudCredential"
```
Comment (paynejacob) [inline tests/framework/extensions/cloudcredentials/config.go]: can we import this from the management client

File: tests/framework/extensions/cloudcredentials/config.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package cloudcredentials
```
Comment (paynejacob) [inline tests/framework/extensions/cloudcredentials/config.go]: nit: rename this file to `cloudcredentials.go`

**Review by paynejacob — APPROVED** [review]

**Review by paynejacob — APPROVED** [review]

**Pull request merged**